compiler/tests/asm.rs

```diff
@@ -1,10 +1,10 @@
 use compiler::verify_asm_string;
-use number::FieldElement;
+use number::GoldilocksField;
 use std::fs;
 
-fn verify_asm(file_name: &str, inputs: Vec<FieldElement>) {
+fn verify_asm(file_name: &str, inputs: Vec<GoldilocksField>) {
```

Comment:
Since this is internal, maybe we should keep it generic and just use GL in the caller side?